mathics/doc/common_doc.py

```diff
@@ -345,9 +345,11 @@ def gather_tests(
             if tests is None:
                 tests = test_collection_constructor()
             tests.tests.append(test)
-        if tests is not None:
-            items.append(tests)
-            tests = None
+
```

Comment:
BTW, the name of this function should not be something like `parse_docstring_to_xmldoc`?

Reply:
I see and agree with your point that the name is a little weird. Let me think about it some more.


I definitely would like to get away from referring to XML-this or that since this isn't anything like XML.